HandyImage.user.js

```diff
@@ -1647,6 +1648,15 @@ function makeworld()
 			i.click();
 			break;
 		}
```

Comment:
see, there's a fall-through there, so right after there your code would get executed for all those sites as well